### PR TITLE
Added device type as a property of the API.

### PR DESCRIPTION
**Pull request by andreasbotsikas**
Added device type as a property of the API.

Jira issue: WP-872

### CODE REVIEW

**Comment by krishnabangalore** [comment]
Please wait don't merge will have to add issupported value

**Comment by mlasak** [comment]
would be good if tests would be provided for this.

**Comment by andreasbotsikas** [comment]
And my indentation is off, so Krishna please fix this too

**Comment by andreasbotsikas** [comment]
In the cc code, the isSupported always returns true no matter what property you ask for... This has to change. http://jira.webinos.org/browse/WP-916

File: webinos/core/api/devicestatus/lib/webinos.rpc.devicestatus.js [hunk 1]
```diff
@@ -27,26 +28,33 @@
...
 		function (params, successCallback) {
-			devicestatusmodule.devicestatus.isSupported(
-				params[0],
-				params[1],
-				function (res) {
-					successCallback(res);
-				}
-			);
+			if (params && params.length == 2 && params[0] == "Device" && params[1] == "type") //If it's the device type
+				successCallback({ aspect:params[0], property:params[1],isSupported:true}); // we support it via config
```
Comment (andreasbotsikas) [inline webinos/core/api/devicestatus/lib/webinos.rpc.devicestatus.js:32]: This is optional as of http://jira.webinos.org/browse/WP-916

**Comment by krishnabangalore** [comment]
This branch has the updated code for the DeviceType property and it is Good to merge and tested on Linux, Windows
can reject the PR for wp-872_devicetype branch since this has the updated code.

One more point to Note:
add/change params in webinos_config.json ex- "devicetype": "laptop" 
 (Your device options to choose - "smartphone", "tablet","ivi","tv","homemediabox","pc","laptop" ) 
and also check if not there then add/change params in (dot webinos) .webinos under user data in the file serviceCache.json "devicetype": "laptop" to run this

**Pull request merged**